docs/web/sdk-components-styling.mdx

```diff
@@ -116,20 +126,189 @@ export default function ComponentWithIcon(props) {
                     secondary={"cloud-download icon"}
                 />
                 <MenuItemSecondaryAction>
-                    // highlight-start
                     <IconButton title="IconExample">
-                        <DynamicIcon src="cloud-download" size="large" />
+                        // highlight-next-line
+                        <DynamicIcon src="cloud-download" fontSize="large" />
+                    </IconButton>
+                </MenuItemSecondaryAction>
+                <MenuItemSecondaryAction>
+                    <IconButton title="IconExample">
+                        // highlight-next-line
+                        <CloudDownload fontSize="large" />
                     </IconButton>
-                    // highlight-end
                 </MenuItemSecondaryAction>
             </MenuItem>
         </LayoutElement>
     );
-}
+};
+
+export default ComponentWithIcon;
 ```
 
 </TabItem>
 <TabItem value="ui">
 <img src={useBaseUrl("img/web-component-with-icon.png")}/>
 </TabItem>
 </Tabs>
+
+## Adding Icons
+
+Both static icons and dynamic SVG icons can be added through the Geocortex Web SDK. Static icons can only be used as React Components, while dynamic icons can also be used in the app config.
+
+### Static Icons
+
+To create a new static SVG icon, create a new `tsx` file that exports the `createSvgIcon` function. This function will create a React Component icon that can be used in Geocortex Web Components.
+
+```tsx title="src/icons/CustomIcon.tsx"
+import createSvgIcon from "@vertigis/web/ui/icons/utils/createSvgIcon";
+import React from "react";
+
+export default createSvgIcon(
+    <path d="M20 11H7.8l5.6-5.6L12 4l-8 8 8 8 1.4-1.4L7.8 13H20v-2z" />
+);
+```
+
+### Dynamic Icons
+
+To create a new dynamic icon, you simply need to create a static icon and then register it with the Geocortex Web SDK using `registry.registerIcon(...)`. Once it is registered, it can be used with the `DynamicIcon` component or in the app config.
```

Comment:
It might be worth explicitly pointing out that custom icons registered this way can be used by any out-of-the-box components, like Button, IWTM, and so on in this paragraph. It's implicit in "... or in app config", and it shows up in your example code, but maybe could be emphasized a bit more. This is typically the most common use case for registering icons, rather than for use in custom components (unless your component is specifically designed to be configurable).

Reply:
Good call! I've explicitly pointed this out now and emphasized that components like Buttons can use icons registered in this manner.